src/apof/restaurants/tests.py

```diff
@@ -0,0 +1,62 @@
+from datetime import time
+from mock import patch, MagicMock
```

Comment:
If it was Python 3 that would be correct, here we have 2.7 - so `mock` is not buildin library. Please move it to correct section.